format.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+CPP_DONE="Source/doom.cpp Source/movie.cpp Source/pfile.cpp Source/player.cpp Source/plrmsg.cpp Source/sound.cpp Source/spells.cpp Source/tmsg.cpp"
+
+for f in $CPP_DONE; do
+	echo "Formatting $f"
+	clang-format -style="{BasedOnStyle: webkit, AlignTrailingComments: true, AllowShortBlocksOnASingleLine: true, AllowShortFunctionsOnASingleLine: None, PointerAlignment: Right}" -i $f
+done
+
+H_DONE="enums.h structs.h"
+
+for f in $H_DONE; do
+	echo "Formatting $f"
+	clang-format -style="{BasedOnStyle: webkit, AlignTrailingComments: true, AllowShortBlocksOnASingleLine: true, AllowShortFunctionsOnASingleLine: None, PointerAlignment: Right, AlignConsecutiveAssignments: true}" -i $f
```

Comment:
I would prefer that we create an actual .clang-format file so that it will be auto discovered by the editor/tools

Reply:
Sure. The main reason I did this way is that I could not find a way to specify the config file for clang-format. (It does not seem possible?)

And `AlignConsecutiveAssignments` is only used for header files, otherwise the configs are identical.

Reply:
Fixed in https://github.com/diasurgical/devilution/pull/390/commits/e41cf1a6f7b74c893920b066157b6838e8ff71e9 and https://github.com/diasurgical/devilution/pull/390/commits/d5672d046141102ea9b83bff721846c7b643af78